scripts/fast-reboot

```diff
@@ -332,6 +341,21 @@ function reboot_pre_check()
     fi
 }
 
+function load_aboot_secureboot_kernel() {
+    local next_image="$IMAGE_PATH/sonic.swi"
+    echo "Loading next image from $next_image"
+    (
+        cd /tmp
```

Comment:
>cd /tmp [](start = 8, length = 7)

Use `mktemp -d` to prevent file confliction?

Reply:
Actually I could remove the block that goes into /tmp since boot0 does not dirty the current directory.
I just did this as a safety mechanism but `mktemp -d` is not useful here.